<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchRegLocationCardDialog</class>
 <widget class="QDialog" name="BatchRegLocationCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Место нахождения учетного документа</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1" colspan="4">
    <widget class="QComboBox" name="cmbDocumentNumber">
     <item>
      <property name="text">
       <string>Не заполнять</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Заполнять кодом пациента</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Заполнять номером документа события</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbDocumentTypeForTracking"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDocumentTypeForTracking">
     <property name="text">
      <string>Вид учетного документа</string>
     </property>
     <property name="buddy">
      <cstring>cmbDocumentLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2" colspan="4">
    <widget class="QTextEdit" name="edtNotesPage"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblDocumentNumber">
     <property name="text">
      <string>Номер учетного документа</string>
     </property>
     <property name="buddy">
      <cstring>cmbDocumentLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblNote">
     <property name="text">
      <string>Примечание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>edtNotesPage</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="btnStart">
     <property name="text">
      <string>Начать</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>326</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="btnRetry">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDocumentLocation">
     <property name="text">
      <string>Место нахождения</string>
     </property>
     <property name="buddy">
      <cstring>cmbDocumentLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Ответственный</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbDocumentLocation"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbDocumentLocation</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>edtNotesPage</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnRetry</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
